.devcontainer/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM mateusamin/protonvpn-cli-ng-build:latest
```

Comment:
This is is a dev container for this project that plugins into a the containers functionality in VS code: https://code.visualstudio.com/docs/remote/containers